tests/kernel/trinity.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Trying to stress kernel with fuzz testing using trinity
+# Maintainer: Anton Smorodskyi<asmorodskyi@suse.com>
+
+use base "opensusebasetest";
+use testapi;
+use utils;
+use strict;
+use serial_terminal 'select_virtio_console';
+use upload_system_log;
+
+sub run {
+    my ($self) = @_;
+    select_virtio_console();
+    my $trinity     = 'trinity-1.8';
+    my $trinity_log = script_output("echo ~$testapi::username/trinity.log");
+    my $syscall_cnt = 1000000;
+    zypper_call('in gcc');
+    assert_script_run("cd /");
+    assert_script_run("wget --quiet " . data_url($trinity) . ".tar.xz -O $trinity.tar.xz");
```

Comment:
I wonder why build it from scratch - there are packages in openSUSE - https://build.opensuse.org/package/show/devel:tools/trinity which are linked to QA:SLE:XY projects as well (i.e. - https://build.suse.de/package/show/QA:SLE12SP4/trinity)

Reply:
Also version 1.8 is a bit old.